tests/tracing/test_pruning.py

```diff
@@ -2037,6 +2043,109 @@ def test_prune_skeleton(
+                        [0, 0, 3, 2, 3, 0, 0],
+                        [0, 9, 0, 0, 0, 9, 0],
+                        [0, 9, 0, 0, 0, 9, 0],
+                        [0, 0, 9, 9, 9, 0, 0],
+                        [0, 0, 0, 0, 0, 0, 0],
+                    ]
+                ),
+            },
+            10,
+            3,
+            "mid",
+            "abs",
+            False,
+            np.asarray(
+                [
+                    [0, 0, 0, 0, 0, 0, 0],
+                    [0, 0, 1, 1, 1, 0, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 0, 3, 1, 3, 0, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 0, 1, 1, 1, 0, 0],
+                    [0, 0, 0, 0, 0, 0, 0],
+                ]
+            ),
+            np.asarray(
+                [
+                    [0, 0, 0, 0, 0, 0, 0],
+                    [0, 0, 1, 1, 1, 0, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 0, 0, 1, 0, 0, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 0, 1, 1, 1, 0, 0],
+                    [0, 0, 0, 0, 0, 0, 0],
+                ]
+            ),
+            np.asarray(
+                [
+                    [0, 0, 0, 0, 0, 0, 0],
+                    [0, 0, 1, 1, 1, 0, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 0, 0, 2, 0, 0, 0],
+                    [0, 3, 0, 0, 0, 3, 0],
+                    [0, 3, 0, 0, 0, 3, 0],
+                    [0, 0, 3, 3, 3, 0, 0],
+                    [0, 0, 0, 0, 0, 0, 0],
+                ]
+            ),
+            np.asarray([9, 2, 9]),  # branch mins
+            np.asarray(
+                [
+                    9,
+                    2,
+                    9,
+                ]
+            ),  # branch_medians : Used in _abs_thresh_idx with threshold of 9
+            np.asarray([9, 2, 9]),  # branch_middles
+            np.asarray([2]),  # abs_thresh_idx : index + 1 in above noted array that are < 10
+            np.asarray([2]),  # mean_abs_thresh_idx
+            np.asarray([2]),  # iqr_thresh_idx
+            False,
+            np.asarray(
+                [
+                    [0, 0, 0, 0, 0, 0, 0],
+                    [0, 0, 1, 1, 1, 0, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 0, 1, 0, 1, 0, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 1, 0, 0, 0, 1, 0],
+                    [0, 0, 1, 1, 1, 0, 0],
+                    [0, 0, 0, 0, 0, 0, 0],
+                ]
+            ),
+            id="bridge (low figure 8)",
+            # marks=pytest.mark.xfail(reason="Not sure middles are correct, arbitrarily takes point left or gith of even
```

Comment:
This looks like a hangover from copy and pasting so regardless of whether the above is correct can likely be removed.

Reply:
have also removed from here :)